src/extension/platform.js

```diff
@@ -29,44 +34,37 @@ export async function getPlatformRootDir(dir = process.cwd()) {
 }
 
 export async function getExtensionsPaths(platformDir) {
-  const paths = await glob(path.join(platformDir, 'extensions', '*', 'app'));
-  console.log(paths);
-  return paths;
+  return await glob(path.join(platformDir, 'extensions', '*', 'app'));
 }
 
-export async function uncommentBuildDir(buildDirectory) {
-  const buildGradlePath = path.join(buildDirectory, 'android', 'build.gradle');
-  let buildGradle = await fs.readFile(buildGradlePath, 'utf-8');
-  buildGradle = buildGradle.replace('//<CLI> buildDir', 'buildDir');
-  await fs.writeFile(buildGradlePath, buildGradle);
-}
-
-export async function preparePlatform(platformDir, { platform, appId, debug = true, excludePackages = ['shoutem.code-push'], production = false, linkLocalExtensions = true }) {
-  const mobileConfig = {
+export async function createMobileConfig(platformDir, { platform, appId, debug = true, excludePackages = ['shoutem.code-push'], production = false, linkLocalExtensions = false, skipNativeDependencies = false, offlineMode = false }) {
```

Comment:
this is ugly. Just rename second param to `opts` and in returned object set default values (e.g. 
```
debug: !!debug,
excludePackages = excludePackages || ['shoutem.code-push']
```
)